### PR TITLE
Include base classes in output

### PR DESCRIPTION
**Pull request by bkoropoff**
Part of resolving https://github.com/mkdocstrings/mkdocstrings/issues/269

### CODE REVIEW

**Review by pawamoy — APPROVED** [review]
I like it, simple and efficient. Thanks!

**Comment by pawamoy** [comment]
I wonder: every class always depend on `object` right? Will object show up for every class' bases?

**Comment by bkoropoff** [comment]
`__bases__` includes only direct base classes, so only `object` will only show up for classes that inherit directly from it with no intermediaries

**Comment by bkoropoff** [comment]
I pushed a fix for a type check failure.  There's also one in docs/macros.py that seems to pre-existing.

**Comment by pawamoy** [comment]
Yes, perfect, thanks, leave the one in the docs, I'll take care of it.

File: src/pytkdocs/objects.py [hunk 1]
```diff
@@ -350,6 +350,18 @@ class Class(Object):
 
     possible_name_properties: List[ApplicableNameProperty] = [NAME_PRIVATE]
 
+    def __init__(self, *args, bases: List[str] = ["object"], **kwargs):
```
Comment (pawamoy) [inline src/pytkdocs/objects.py:353]: Just saw this (not sure it was there earlier, sorry for missing it): I don't think it has any impact on the current code, but mutable defaults are not considered good-practice.
If `bases` is not specified when creating instances of `Class`, all these instances will share the same `["object"]` list.
Could you instead default to None and create the `["object"]` list in the body of the function?

Comment (bkoropoff) [reply src/pytkdocs/objects.py:353]: Ah, yeah, good idea

**Comment by bkoropoff** [comment]
Is this ready to merge @pawamoy ?

**Comment by pawamoy** [comment]
Thanks again @bkoropoff!

**Pull request merged**